specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,222 @@
+| Playing     | A glyph that indicates media or content is currently playing.                                               | ![Screenshot](badge-playing.png)     |
+| Unavailable | A glyph that shows the user or a service is not currently available or offline.                             | ![Screenshot](badge-unavailable.png) |
+
+These glyphs are system-provided and standardized, ensuring a consistent look and feel across
+different applications that use them. By using these glyphs, developers can convey specific states
+or notifications to users in a visually intuitive manner.
+
+> Note: Only system-provided badge images can be used.
+
+# API Details
+
+> Note: all of this new WinAppSDK API is to support Badge Notifications in WindowsAppSdk
+
+```c++ (but really MIDL3)
+namespace Microsoft.Windows.BadgeNotifications
+{
+    // Set of predefined glyphs that can be used to represent various statuses or notifications on an application's badge
+    enum BadgeNotificationGlyph
+    {
+        None, // No glyph. A blank tile appears in the badge.
+        Activity, // A glyph representing an activity
+        Alert, // A glyph representing an alert
+        Alarm, // A glyph representing an alarm
+        Attention, // A glyph representing attention
+        Available, // A glyph representing availability
+        Away, // A glyph representing being away
+        Busy, // A glyph representing being busy
+        NewMessage, // A glyph representing a new message
+        Paused, // A glyph representing being paused
+        Playing, // A glyph representing playing
+        Unavailable, // A glyph representing being unavailable
+        Error, // A glyph representing an error
+    };
+
+    runtimeclass BadgeNotification
+    {
+        // Constructs a BadgeNotification with a numeric badge value.
+        // This constructor initializes the BadgeNotification with a number that will be displayed on the badge.
+        // @param num The numeric value to be displayed on the badge.
+        BadgeNotification(UInt32 notificationCount);
+
+        // Constructs a BadgeNotification with a glyph value.
+        // This constructor initializes the BadgeNotification with a predefined glyph from the BadgeNotificationGlyph enum.
+        // @param value The BadgeNotificationGlyph representing a system-provided glyph to be displayed on the badge.
+        BadgeNotification(BadgeNotificationGlyph value);
+
+        // Gets or sets the time after which a Badge Notification should not be displayed.
+        Windows.Foundation.DateTime Expiration;
+    }
+
+    // The manager class which encompasses all Badge Notification API Functionality
+    runtimeclass BadgeNotificationManager
+    {
+        // Gets a Default instance of a BadgeNotificationManager
+        static BadgeNotificationManager Current{ get; };
+
+        // Applies a change to the badge's glyph or number.
+        void Update(BadgeNotification notification);
```

Comment:
Would it be better to get rid of the BadgeNotification entirely and just have three methods
* manager.SetBadgeAsGlyph(BadgeNotificationGlyph value, DateTime expiration)
* manager.SetBadgeAsNumber(UInt32 value, DateTime expiration)
* manager.ClearBadge()